docs/local-chain.md

```diff
@@ -0,0 +1,125 @@
+# How to run a local chain
+
+To run a custom chain three things are needed
+- a node
+- genesis
+- at least one set of baker credentials
+
+The instructions for building your own node are in [a separate
+document](https://github.com/Concordium/concordium-node/tree/main/concordium-node#building-the-node).
+
+Here we describe how to build a genesis block and start the node(s) with the
+given genesis.
+
+We have a
+[genesis-creator](https://github.com/Concordium/concordium-misc-tools/tree/main/genesis-creator)
+tool that can be used to create genesis blocks configured to specific scenarios.
+Please see the README in the linked repository for how to build the tool.
```

Comment:
The readme doesn't actually tell you how to _build_ the tool. Perhaps we should add that.
More importantly, the readme tells you how to _use_ the tool.
```suggestion
Please see the README in the linked repository for how to build and use the tool.
```